<!--Launches a race_tabletop_segmentation node. Remaps all parameters.-->
<launch>
    <node ns="race_tabletop_segmentation" pkg="race_tabletop_segmentation" name="race_tabletop_segmentation_node" type="race_tabletop_segmentation_node" output="screen">
	<!--topic remapping-->
	<remap from="cloud_in" to="/camera/depth_registered/points" />
	<remap from="segmentation_srv" to="segmentation_srv" />
	<remap from="segmentation_markers" to="segmentation_markers" />

	<!--<param name="clustering_voxel_size" value="$(arg tabletop_segmentation_clustering_voxel_size)" />-->
	<!--<param name="inlier_threshold" value="300" />-->
	<!--<param name="plane_detection_voxel_size" value="0.01" />-->
	<!--<param name="cluster_distance" value="0.1" />-->
	<!--<param name="min_cluster_size" value="10" />-->
	<!--<param name="flatten_table" value="$(arg flatten_table)" />-->
	<!--<param name="table_padding" value="$(arg table_padding)" />-->

	<!-- processing and filtering frame -->
	<!-- all clouds converted to and processed in base link frame -->
	<!--<param if="$(arg tabletop_segmentation_convert_to_base_link)" name="processing_frame" value="base_link" />-->
	<!--<param if="$(arg tabletop_segmentation_convert_to_base_link)" name="up_direction" value="1.0" />-->
	<!--<param if="$(arg tabletop_segmentation_convert_to_base_link)" name="z_filter_min" value="0.35" />-->
	<!--<param if="$(arg tabletop_segmentation_convert_to_base_link)" name="z_filter_max" value="1.0" />-->
	<!--<param if="$(arg tabletop_segmentation_convert_to_base_link)" name="y_filter_min" value="-1.2" />-->
	<!--<param if="$(arg tabletop_segmentation_convert_to_base_link)" name="y_filter_max" value="1.2" />-->
	<!--<param if="$(arg tabletop_segmentation_convert_to_base_link)" name="x_filter_min" value="0.3" />-->
	<!--<param if="$(arg tabletop_segmentation_convert_to_base_link)" name="x_filter_max" value="1.0" />-->
	<!--<param if="$(arg tabletop_segmentation_convert_to_base_link)" name="table_z_filter_min" value="-0.5" />-->
	<!--<param if="$(arg tabletop_segmentation_convert_to_base_link)" name="table_z_filter_max" value="-0.01" />		-->

    </node>
</launch>
